### ISSUE TITLE
Refine log parsing in gathering transaction details

### ISSUE THREAD

**Issue by jubalm** [DarkFlorist/lunaria#90]
I'm still a bit uncomfortable with this.  A single transaction can have multiple transfer events.  For example, some ERC20 tokens will transfer a fee to the token creator every time a token is transferred, as well as transferring to the designated recipient.  In this code we are blindly assuming that the first transfer event (log) is the one we care about, but there is no guarantee in the standard that that is the case.

Do we know who the expected sender/recipient is when we are calling this?  If so, we should filter on those.  If we don't, we may want to consider just presenting all of the events to the user rather than just the first one.

Note: If it is a lot of work to do these, we can file issues for them and address them in a separate PR.

_Originally posted by @MicahZoltu in https://github.com/DarkFlorist/lunaria/pull/89#discussion_r1140943571_

**Comment by jubalm** [comment]
for now scan only transactions coming from the active account

**Comment by jubalm** [comment]
Log parsing is no longer necessary after #111
